pkg/image/sif.go

```diff
@@ -12,25 +12,25 @@ import (
 	specs "github.com/opencontainers/runtime-spec/specs-go"
 )
 
-type sif struct {
+type SIF struct {
 }
 
-// SifFromSandbox converts the sandbox, s, to a SIF file
-func SifFromSandbox(s *Sandbox) *sif {
-	return &sif{}
+// SIFFromSandbox converts the sandbox, s, to a SIF file
+func SIFFromSandbox(s *Sandbox) *SIF {
+	return &SIF{}
 }
 
-// SifFromPath returns a sif object of the file located at path
-func SifFromPath(path string) *sif {
-	return &sif{}
+// SIFFromPath returns a SIF object of the file located at path
+func SIFFromPath(path string) *SIF {
```

Comment:
How about an io.ReadSeeker as the input? Adds flexibility an allows us to minimize file handle open/closes.